sagemaker_neo_compilation_jobs/mxnet_mnist/mxnet_mnist_neo.ipynb

```diff
@@ -194,16 +195,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import io\n",
-    "import numpy as np\n",
-    "def numpy_bytes_serializer(data):\n",
-    "    f = io.BytesIO()\n",
-    "    np.save(f, data)\n",
-    "    f.seek(0)\n",
-    "    return f.read()\n",
-    "\n",
-    "predictor.content_type = 'application/vnd+python.numpy+binary'\n",
-    "predictor.serializer = numpy_bytes_serializer"
+    "from sagemaker.serializers import NumpySerializer\n",
+    "npy_serializer = NumpySerializer()\n",
+    "predictor = compiled_model.deploy(initial_instance_count = 1,\n",
```

Comment:
is the SDK fixed now to use the correct inference container URI?

Reply:
yes